abstractions/php/src/Serialization/ParseNode.php

```diff
@@ -98,6 +99,11 @@ public function getEnumValue(string $targetEnum): ?Enum;
      */
     public function getByteValue(): ?Byte;
 
+    /**
+     * Get a Stream from node.
+     * @return StreamInterface|null
+     */
+    public function getBinaryContent(): ?StreamInterface;
```

Comment:
I'm not sure whether this was intended to be part of this PR or got commit by mistake?